data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/events/EventSubscriber.java

```diff
@@ -34,26 +36,32 @@
   private final Runnable closeCallback;
   private final int maxPendingEvents;
   private final AtomicBoolean processingQueue;
+  private final TimeProvider timeProvider;
+  private UInt64 lastEventSentSeconds;
```

Comment:
I suspect this isn't thread safe - multiple threads are accessing it so it probably needs to be an `AtomicReference` and need to be a bit clever in the `keepAlive` method to handle two threads checking if they should send a comment at the same time.